app/views/users/index.html.erb

```diff
@@ -1,20 +1,24 @@
 
 <h1>User Listing</h1>
 
-<table>
-  <tr>
-    <th>Full Name</th>
-    <th>Email Address</th>
-    <th>Phone Number</th>
-    <th>Join Date</th>
-  </tr>
+<table class="table table-striped">
```

Comment:
Per feature requirements this needs to list all of the users, so this should be on the `index` page, where you will loop through users
```erb
# users/index.html.erb
<% @users.each do |user| %>
   # table row
<% end %>
```